buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring_1_26_0.groovy

```diff
@@ -23,53 +23,54 @@ package org.apache.beam.gradle
  */
 class GrpcVendoring_1_26_0 {
 
-  static def guava_version = "26.0-jre"
-  static def protobuf_version = "3.11.0"
+  static def alpn_api_version = "1.1.2.v20150522"
+  static def bouncycastle_version = "1.54"
+  static def compress_lzf_version = "1.0.3"
+  static def conscrypt_version = "1.3.0"
+  static def google_auth_version = "0.18.0"
   static def grpc_version = "1.26.0"
   static def gson_version = "2.8.6"
+  static def guava_version = "26.0-jre"
+  static def jboss_marshalling_version = "1.4.11.Final"
+  static def jboss_modules_version = "1.1.0.Beta1"
+  static def jzlib_version = "1.1.3"
+  static def log4j_version = "2.13.3"
```

Comment:
Notice that vendored grpc dependencies can have different versions so they don't depend on Beam main module file, so versions are defined independently.